bad.R

```diff
@@ -0,0 +1,10 @@
+fun = function(one)
+{
+  one.plus.one <- oen + 1
+  four <- newVar <- matrix(1:10,nrow = 2)
+  four[ 1, ]
+  txt <- 'hi'
+  three <- two+ 1
+  if(txt == 'hi') 4
```

Comment:
What should happen with 4 and 5 if txt == "hi"?

Reply:
Good question. I'm not sure